script.js

```diff
@@ -1,4 +1,153 @@
+// computer random chooses scissors, paper, or stone
+var computerChoice = function () {
+  const game = ["scissors", "paper", "stone"];
+
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+
+  var computerhand = game[randomInteger];
+  return computerhand;
+};
+
+var inputName = function (name) {
+  return `Hello ${name}, please input 'scissors' 'paper' or 'stone' to start playing the game.`;
+};
+
+var getIcon = function (hand) {
+  if (hand == "scissors" || hand == "reversed scissors") {
+    return "✂️";
+  }
+  if (hand == "paper" || hand == "reversed paper") {
+    return "🗒";
+  }
+  if (hand == "stone" || hand == "reversed stone") {
+    return "🪨";
+  }
+};
+
+// normal game mode
+var playNormalGame = function (playerHand, computerHand) {
+  if (
+    (playerHand == "scissors" && computerHand == "paper") ||
+    (playerHand == "paper" && computerHand == "stone") ||
+    (playerHand == "stone" && computerHand == "scissors")
+  ) {
+    return "win";
+  }
+  if (
+    (playerHand == "scissors" && computerHand == "stone") ||
+    (playerHand == "paper" && computerHand == "scissors") ||
+    (playerHand == "stone" && computerHand == "paper")
+  ) {
+    return "lose";
+  }
+  if (
+    (playerHand == "scissors" && computerHand == "scissors") ||
+    (playerHand == "paper" && computerHand == "paper") ||
+    (playerHand == "stone" && computerHand == "stone")
+  ) {
+    return "draw";
+  } else {
+    return "error";
+  }
+};
+
+// reversed game mode
```

Comment:
splitting your two game modes as functions outside of the main function and using a global variable to call each one makes your code cleaner more readable. Really well done!